### DISCUSSION TITLE
Prefilter records on the grid

### DISCUSSION THREAD

**Discussion by rcmsoftwaredevelopment** [serenity-is/Serenity#7011] [Help] [answered]
Hi all, I have been away for a few years and just coming back to this awesome project.  I'm trying to prefilter a grid but when looking at the code on https://demo.serenity.is/BasicSamples/GridFilteredByCriteria.  It works, but it seems to clear down my joined tables in the grid.  What am I missing?

**Comment by luongnv6** [comment, marked as answer]
pls show your code

**Reply by rcmsoftwaredevelopment** [reply]
Resolved - Me being a noob
